### DISCUSSION TITLE
ロケットの魔法棒を確定で入手できる機会を設けてほしい

### DISCUSSION THREAD

**Discussion by Nupepo** [hengband/hengband#974] [Ideas]
現行のver3.0.0alpha20にて、
ロケットの魔法棒はダンジョンの床落ち生成品やモンスタードロップ、
果てはブラックマーケットを巡ることでしか最初の1本目を見つけることができません

下記のような魔法道具確定手段があるように、何かしらの確定入手機会（アリーナ・ダンジョン主討伐・クエスト……etc）を設けて頂ければ、
魔道具術師や観光客、MP切れしやすいメイジ職のお守りになり、ゲーム進行がスムーズになります

ロケット魔法棒それ自体が非常に強力すぎるアイテム故に確定入手できる機会の扱いが繊細にならざるを得ないでしょうが、
ぜひ……ぜひご検討のほど宜しくお願い申し上げます

以下現行での強力な魔法道具アイテム確定入手機会
魔力の嵐の杖……火山の主『シューティングスター』を倒す
衝撃の魔法棒……アリーナの29戦目ナイトウォーカーを倒す
分解の魔法棒……アリーナの36戦目超人ロックのクローンを倒す

**Comment by Nupepo** [comment]
ものすご～く個人的なお願いとしては、
辺境の宿屋クエストで古い城の次にサイバーデーモンと一騎討するという高難度クエストを追加しただければ…と

DooMのE2M8にあやかってクエストフロア内にロケット棒が落ちておりそれを拾ってサイバーデーモンを倒すというクエストです
クエスト報酬として壊滅のロッド（BFG9000）が手に入るととても楽しそうです

古い城という有用な★を得るクエストの後に追加することで、
クエスト放棄してロケ棒狙いという行動に多少リスクを持たせられたらという狙いもあります

**Reply by iks3** [reply]
ロケット棒は最終兵器ですので確定で落ちるのはどうかなあといった感じですので、実現性は低いと思います。
ただ、Oremorjが確定で落とすのはどうかという話はあったので、それはそれっぽいので検討してみます。
